t/gcp_project_services/environment-main.tf

```diff
@@ -0,0 +1,16 @@
+locals {
+  project_id = "{{.project_id}}"
+}
+
+resource "google_project_service" "services" {
+  for_each = toset([
+    {{/* Replace comma with quote-comma-newline-fourspaces. Because TODO */ -}}
+    "{{replaceAll .services "," "\",\n    \""}}",
```

Comment:
Can we ensure these are sorted lexographically?

Reply:
Sure, I can add sorting here https://github.com/abcxyz/abc/blob/main/templates/commands/render_action.go#L122 in a later PR.